docs/openshift_ci.md

```diff
@@ -251,6 +251,92 @@ Credentials are provided to `create_pr.py` with following environmental variable
 - `REPO` - repository to work against (push branch and create PR)
 - `ORG` - repository's organization
 
+## CI Troubleshooting
+
+### sosreport Utility
+CI jobs spinning up MicroShift cluster run the [sosreport](https://github.com/sosreport)
+utility to collect the information about the job runtime environment, using the
+`container`, `network` and `microshift` plugins.
+
+The report is created before deprovisioning the CI infrastructure and the output
+of the utility is stored together with the CI artifacts, to be accessed under the
+`artifacts/<test_name>/openshift-microshift-infra-sos-aws` directory.
+
+The top-level `build-log.txt` file contains the runtime output of the `sosreport`
+utility, while another `artifacts` sub-directory contains the actual `sosreport`
+output archives.
+
+The `sosreport` archives need to be downloaded and unpacked using `tar xf` command
+to access the collected information.
+
+### Node Performance
+CI jobs spinning up MicroShift cluster install the [pcp-zeroconf](https://access.redhat.com/articles/3115691)
+package intended to simplify the installation and configuration of the most commonly
+needed [Performance Co-Pilot](https://pcp.io/documentation.html) features.
+
+The performance collection starts right after the MicroShift RPM packages are installed
+and the results file is generated before deprovisioning the CI infrastructure. The file
+is stored together with the CI artifacts, to be accessed under the `artifacts/<test_name>`
+directory.
+
+The `openshift-microshift-infra-pcp` sub-directory contains the `build-log.txt` file with
+the package installation logs, while the `openshift-microshift-infra-pmlogs` sub-directory
+contains the actual performance collection results.
+
+It is necessary to drill down to `openshift-microshift-infra-pmlogs/artifacts/<ci_hostname>`
+directory to reach the `pmlogger.log` collection logs and the `yyyymmdd.hh.mm.{0,index,meta}`
+collection data.
+
+Install the `gsutil` utility necessary for downloading the collection data directory contents.
```

Comment:
Just one nit, I would mention this as recommended but not mandatory, but up to you.

Reply:
Good point. Rephrased the sentense to denote that it's one of the tools that can be used.